includes/class-apple-news.php

```diff
@@ -1,105 +1,184 @@
 <?php
 /**
+ * Publish to Apple News Includes: Apple_News class
+ *
+ * Contains a class which is used to manage the Publish to Apple News plugin.
+ *
+ * @package Apple_News
+ * @since 0.2.0
+ */
+
+/**
  * Base plugin class with core plugin information and shared functionality
  * between frontend and backend plugin classes.
  *
- * @author  Federico Ramirez
- * @since   0.2.0
+ * @author Federico Ramirez
+ * @since 0.2.0
```

Comment:
Unfortunately I think we need to not change these values and return them to their previous names and variables.

Because they were public/protected, in theory it's possible that an advanced developer could be relying on them in custom code built against the plugin. Unfortunately I think we have to live with them as-is.